node-red/cloudant-to-dashdb.json

```diff
@@ -77,7 +77,7 @@
     "id": "af2d1ef.6818ae",
     "type": "cloudant",
     "z": "",
-    "host": "education.cloudant.com",
-    "name": "education account"
+    "host": "examples.cloudant.com",
+    "name": "examples account"
   }
 ]
```

Comment:
Just noticed this one is missing a newline.

Reply:
Done in 76f4de5.